styles.css

```diff
@@ -0,0 +1,193 @@
+/* This is the navbar section */
+body {
+    margin: 0;
+    padding: 0;
+}
+.font {
+    font-family: 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+    font-weight: lighter;
+}
+.container {
+    background-attachment: scroll;
+    background-image: url(https://web.archive.org/web/20140301004302im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg);
+    background-position: 50% 100%;
+    background-repeat: no-repeat;
+    position: relative;
+}
+.c-nav {
+    position: relative;
+    margin: 0 auto;
+    top: 80px;
+    width: 80%;
+}
+.nav-list {
+    background-image: linear-gradient(rgb(122, 120, 120) 51%, rgb(100, 98, 98) 51%, rgb(129, 129, 129));
```

Comment:
Good Job using linear-gradient here, consider adding similar gradient for hover effects on these buttons.